stories/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Can we rename this component to ExampleButton so that we know that it was created only as an example implementation for storybooks?

Reply:
All the stories inside `stories` directory are sample stories.
We can remove it as a whole ,if it's not required .
When running storybook locally , you can see they all come under Example dropdown. (see Image in pr for reference)
Do let me know what would be better to do.

Reply:
I feel ExampleButton will still be better. Because in the IDE when somebody is trying to open the Button component of the app, both the components will show up and that might confuse the dev a little